build/patch-charts.sh

```diff
@@ -169,20 +177,20 @@ function patchGalley() {
 \1  containerPort: 15017/' $deployment
 
   # always istiod
-  sed_wrap -i -e '/{{- if eq .Values.revision ""}}/,/{{- end }}/ { /istiod/! d }' $deployment
+#  sed_wrap -i -e '/{{- if eq .Values.revision ""}}/,/{{- end }}/ { /istiod/! d }' $deployment
 
   # multitenant
-  echo '
```

Comment:
didn't we remove this as part of https://github.com/maistra/istio-operator/pull/873 ? 

actually, it looks like that change was never cherry-picked

Reply:
Yeah, I Told @jewertow not to cherry-pick it, because I knew this would be in a different place in 2.2.